src/components/App/App.js

```diff
@@ -30,40 +30,17 @@ function App() {
   return (
     <Router>
       <div className="App" data-testid="App">
-        <nav>
-          <ul>
-            {user ? (
-              <>
-                <li>
-                  <Link to="/">Dashboard</Link>
-                </li>
-                {user.isAdmin ? (
-                  <li>
-                    <Link to="/accounts/new">Add Account</Link>
-                  </li>
-                ) : null}
-                <li>
-                  <Link to="/" onClick={logOut}>
-                    Log out
-                  </Link>
-                </li>
-              </>
-            ) : null}
-          </ul>
-        </nav>
-        <Container>
-          <Route path="/" exact component={withRestrictedAccess(Dashboard)} />
-          <Route
-            path="/accounts/new"
-            component={withRestrictedAccess(AddStaff, true, user)}
-          />
-          <Route
-            path="/login/"
-            render={props => <Login {...props} setUser={setUser} />}
-          />
-          <Route path="/addfarmer" component={AddFarmer} />
-          <ToastContainer position="top-right" />
-        </Container>
+        <Route path="/" exact component={withRestrictedAccess(Dashboard)} />
```

Comment:
Please, add the `<Container></Container>` wrapping from line 33 to line 43! This container is used to put a healthy margins in all the pages 👍